packages/kit/src/runtime/server/endpoint.js

```diff
@@ -92,7 +96,7 @@ export async function render_endpoint(event, mod) {
 		}
 	}
 
-	return new Response(normalized_body, {
+	return new Response(method !== 'head' ? normalized_body : '', {
```

Comment:
strictly speaking i think this should be `undefined`, though i'm not sure there's an observable distinction

```suggestion
	return new Response(method !== 'head' ? normalized_body : undefined, {
```

Reply:
done, had a test fail locally with a redirect loop on first run but that seems to have been a fluke, second run passed